.github/workflows/test-build.yml

```diff
@@ -61,10 +61,60 @@ jobs:
         env:
           MODULE: ${{ matrix.module }}
 
+  build-base-image:
```

Comment:
Since the build time should be fairly fast once the docker files are compressed into a single one, you should be able to define a single job here: `build-docker` and build and push all the containers within that job.